js/Snake.js

```diff
@@ -0,0 +1,150 @@
+'use strict';
+
+class SnakeHead extends Square {
+  constructor(position) {
+    super(position, 'limegreen');
+  }
+}
+
+class SnakeBody extends Square {
+ constructor(position) {
+    super(position, 'darkgreen');
+  } 
+}
+
+class Snake extends Drawable {
+  constructor(position, size) {
+    super();
+
+    this.size = size;
+
+    this.growLeft = 0;
+
+    this.head = new SnakeHead(position);
+    this.body = this.getBody();
+
+    this.direction = 'left';
+    this.nextDirection = null;
+  }
+
+  getBody() {
+    const body = [];
+
+    for (let i = 0; i < this.size - 1; i++) {
+      body.push(new SnakeBody(new Point(this.head.position.x + i + 1, this.head.position.y)));
+    }
+
+    return body;
+  }
+
+  grow(n) {
+    this.growLeft += n;
+    for (let i = 0; i < n; i++){
```

Comment:
When the shake eats a fruit, it should grow by a certain amount of body parts during the next frames. A single body part will be added the next time the snake moves, and so on until `growLeft` is zero. Hence the creation of a body part must be done in the `move` method.